stelter/StelterAPI
# Global/v2 linkify update to not link inside script, style, noscript tags

To test:

See the script at the bottom of the page that is being incorrectly linkified (http://stelter-devdss.ingeniuxondemand.com/placeofhope-myplannedgift). Inspect and you will see this:

``` 
<script>
delete s;
s={}
s.getQueryParam=function(){return ""}
s.t=function(){return ""}
<a href="http://s.tl" class="linkified" target="_blank">s.tl</a>=function(){return ""}
</script>
```
Review:
- [ ] Look at the fix in preview (https://stelter-dev.ingeniuxondemand.com/dssPreview/x392830.xml) to see that the s.tl function in that script is now correctly not being linkified. 
- [ ] Check that the two unlinked test links in the intro copy are correctly linkified. 
- [ ] Check on the wills page that the email address in next steps is NOT linkified (it should have contactEmAddr class). Also check on the contact page email address for the same thing.
